### DISCUSSION TITLE
Breadcrumb

### DISCUSSION THREAD

**Discussion by Febakke** [digdir/designsystemet#1824] [Components]
## Related information and insight
- Carbon: https://carbondesignsystem.com/components/breadcrumb/usage/
- Fælles designsystem: https://designsystem.dk/komponenter/broedkrumme/

## Specification and requirements
- Should come in 3 sizes: `Small`, `medium`, `large`. 
- Should use the link-component. 
- Design for small and large screen sizes. 
- A separator should provide clearly distinguishes between each page.

Issue: #1095

**Comment by mrosvik** [comment]
The component is created and published: https://next.storybook.designsystemet.no/?path=/docs/komponenter-breadcrumbs--docs
